workflows_api/runtime/src/main.py

```diff
@@ -30,7 +30,7 @@
         "url": "https://www.apache.org/licenses/LICENSE-2.0.html",
     },
     contact={"url": "https://github.com/NASA-IMPACT/veda-backend"},
-    root_path=settings.workflow_root_path,
+    root_path=settings.workflow_root_path or "/api/workflows",
```

Comment:
A little nervous about this since we usually keep defaults in config. But seems that there isn't a simple fix to handle empty stings